doc/classes/Animation.xml

```diff
@@ -600,9 +600,6 @@
 		<constant name="INTERPOLATION_CUBIC" value="2" enum="InterpolationType">
 			Cubic interpolation.
```

Comment:
Maybe this should mention it's with uniform time now?

Reply:
I think fixed times cubic interpolation is a more special case.